File: 0412-fizz-buzz/0412-fizz-buzz.go
func fizzBuzz(n int) []string {
    var answer [] string
    
    for i := 1; i <= n; i++ {
        if i%3 == 0 && i%5 == 0 {
            answer = append(answer, "FizzBuzz")
        } else if i%3 == 0 {
            answer = append(answer, "Fizz")
        } else if i%5 == 0 {
            answer = append(answer, "Buzz")
        } else {
            // answer = append(answer, fmt.Sprintf("%v",i))
            answer = append(answer, strconv.Itoa(i))
        }
    }
    
    return answer
}

File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
# # python ternary operator
# min = "a is minimum" if a < b else "b is minimum"

        answer = []
        
        for i in range(1,n+1):
            
            answer.append("FizzBuzz" if i % 3 == 0 and i % 5 == 0 else "Fizz" if i % 3 == 0 else "Buzz" if i % 5 == 0 else str(i))

            # if i%3 == 0 and i%5 == 0 :
            #     answer.append("FizzBuzz")
            # elif i%3 == 0:
            #     answer.append("Fizz")
            # elif i%5 == 0:
            #     answer.append("Buzz")
            # else:
            #     answer.append(str(i))
    
        return answer